subsys/bluetooth/host/Kconfig.gatt

```
@@ -141,6 +141,12 @@ config BT_GATT_AUTO_DISCOVER_CCC
 	  This option enables support for GATT to initiate discovery for CCC
 	  handles if the CCC handle is unknown by the application.
 
+config BT_GATT_AUTO_UPDATE_MTU
+	bool "Support to automatic MTU update"
```

- "Support to" sounds ungrammatical. It should be just "Support".
- I would like to drop "Support" because it makes the sentence unnecessarily vague.
- We should specify which MTU value we refer to, like "ATT MTU".

```suggestion
	bool "Automatically send the ATT MTU exchange request on connect"
```

---

I would even say `bool "Automatically send ATT MTU exchange request on connect"` (drop "the").